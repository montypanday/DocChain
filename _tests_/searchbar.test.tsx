import * as React from "react";
import { SearchBar } from "../clientapp/components/featurebar/searchbar";
import * as renderer from "react-test-renderer";

test("Link changes the class when hovered", () => {
    const changeHandler = jest.fn();
    const searchHandler = jest.fn();
    const component = renderer.create(
        <SearchBar
            changeHandler={changeHandler}
            searchHandler={searchHandler}>
        </SearchBar>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});