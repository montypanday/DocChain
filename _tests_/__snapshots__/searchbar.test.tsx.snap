// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="srch-bar-container"
>
  <div
    className="input-group"
  >
    <input
      className="form-control srch-bar"
      onChange={[MockFunction]}
      onKeyPress={[Function]}
      placeholder="Search for files and folders"
      type="text"
    />
    <span
      className="input-group-btn"
    >
      <button
        className="btn btn-default btn-line"
        onClick={[MockFunction]}
        type="button"
      >
         Search
      </button>
    </span>
  </div>
</div>
`;
